internal/statestore/redis_indices.go

```diff
@@ -35,8 +33,14 @@ import (
 // Filters are strings look up that attribute/value pair.
```

Comment:
Update the comment above this function needs to change to reflect the tags instead of boolean.

Reply:
done.